src/modules/Dropdown/Dropdown.js

```diff
@@ -82,121 +66,591 @@ export default class Dropdown extends Component {
         PropTypes.number,
       ]),
     ]),
-    options: PropTypes.arrayOf(PropTypes.shape({
-      value: PropTypes.oneOfType([
-        PropTypes.string,
-        PropTypes.number,
-      ]),
-      text: PropTypes.oneOfType([
+
+    /** Must match one of the option values. */
+    defaultValue: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.number,
+      PropTypes.arrayOf([
         PropTypes.string,
         PropTypes.number,
       ]),
-    })),
-  }
+    ]),
+
+    /** Placeholder text. */
+    placeholder: PropTypes.string,
+
+    /** Name of the hidden input which holds the value. */
+    name: PropTypes.string,
+
+    // ------------------------------------
+    // Callbacks
+    // ------------------------------------
+
+    /** Called with the React Synthetic Event on Dropdown blur. */
+    onBlur: PropTypes.func,
+
+    /** Called with the current value on change. */
+    onChange: PropTypes.func,
+
+    /** Called with the React Synthetic Event on Dropdown click. */
+    onClick: PropTypes.func,
+
+    /** Called with the React Synthetic Event on Dropdown focus. */
+    onFocus: PropTypes.func,
+
+    // ------------------------------------
+    // Style
+    // ------------------------------------
+
+    /** Format the Dropdown to appear as a button. */
+    button: PropTypes.bool,
+
+    /** Additional classes added to the root element. */
+    className: PropTypes.string,
+
+    /** Format the dropdown to only take up as much width as needed. */
+    compact: PropTypes.bool,
+
+    /** Format the dropdown to only take up as much width as possible. */
+    fluid: PropTypes.bool,
+
+    /** Display the menu as detached from the Dropdown. */
+    floating: PropTypes.bool,
+
+    inline: PropTypes.bool,
+    labeled: PropTypes.bool,
+    linkItem: PropTypes.bool,
+
+    /** Allow selecting multiple options. */
+    multiple: PropTypes.bool,
+
+    /** Use a detached menu that is pointing to the Dropdown. */
+    pointing: PropTypes.oneOfType([
+      PropTypes.bool,
+      PropTypes.oneOf(_meta.props.pointing),
+    ]),
+
+    /** Make the dropdown options searchable. */
+    search: PropTypes.bool,
+
+    /** The text displayed in the dropdown, usually for the active item. */
+    text: PropTypes.string,
 
-  static defaultProps = {
-    icon: 'dropdown',
+    // TODO 'searchInMenu' or 'search='in menu' or ???  How to handle this markup and functionality?
```

Comment:
aye, thanks. this is a reminder to handle the [search in menu](http://semantic-ui.com/modules/dropdown.html#search-in-menu) dropdown.